pkg/objstore/objtesting/foreach.go

```diff
@@ -55,7 +54,7 @@ func ForeachStore(t *testing.T, testFn func(t testing.TB, bkt objstore.Bucket))
 		testutil.Ok(t, err)
 
 		ok := t.Run("aws s3", func(t *testing.T) {
-			defer leaktest.CheckTimeout(t, 10*time.Second)()
+			// TODO(bplotka): Add leaktest when we fix potential leak in minio.
```

Comment:
Can you add more context to this (more words or link to github issue)?

Reply:
mhmh damn, I investigated GCS one, not minio one yet... Can I do it in next PRs?